app/helpers/blacklight/component_helper_behavior.rb

```diff
@@ -37,6 +37,10 @@ def document_actions(document, options: {})
       filter_partials(blacklight_config.view_config(:show).document_actions, { document: document }.merge(options)).map { |_k, v| v }
     end
 
+    def header_actions(document, options: {})
+      filter_partials(blacklight_config.view_config(:show).header_actions, { document: document }.merge(options)).map { |_k, v| v }
```

Comment:
Is it possible to do?
```suggestion
      filter_partials(blacklight_config.view_config(:show).header_actions, { document: document }.merge(options)).values
```

Reply:
Nope causes an error. Something about how the ActionsComponent is set up doesn't like it.